packages/jaeger-ui/src/components/TracePage/keyboard-shortcuts.js

```diff
@@ -64,6 +67,8 @@ export const kbdMappings = {
   expandAll: '[',
   collapseOne: 'p',
   expandOne: 'o',
+  searchSpans: 'ctrl+b',
```

Comment:
For this, why not use `meta` ?

Reply:
The keyboard handler adds the inputted character to the input. Meta+b outputs a character, while ctrl+b does not.

A work around to use meta is to remove all instances of the '∫' char from the input string, but I didn't want to clutter the code. I tried having the textFilter shortened by 1 char after the event goes off, but unfortunately the input box receives the char after the completion of the event.